authorization/invitation/service/invitation_service.go

```diff
@@ -421,16 +440,45 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 
 			err = s.Repositories().IdentityRoleRepository().Create(ctx, ir)
 			if err != nil {
-				return resourceID, err
+				return resourceID, redirectPath, err
 			}
 		}
 
+		redirectPath, e := s.getRelativeSpacePath(ctx, inviteToResource.ResourceID)
```

Comment:
please use `e` instead of `err`, the `return...` code below will not return the actual error ;)